src/FsCodec.SystemTextJson/FsCodec.SystemTextJson.fsproj

```diff
@@ -11,12 +11,14 @@
   <ItemGroup>
     <Compile Include="JsonSerializerElementExtensions.fs" />
     <Compile Include="Utf8JsonReaderExtensions.fs" />
-    <Compile Include="JsonOptionConverter.fs" />
     <Compile Include="JsonRecordConverter.fs" />
+    <Compile Include="JsonOptionConverter.fs" />
+    <Compile Include="Pickler.fs" />
```

Comment:
Should these be here?

Reply:
hm, UnionConverter shouldnt be, will double check

Reply:
turns out UnionConverter is where those helpers make sense to house when that converter is ported...